docs/overview/zio_prelude_schema.md

```diff
@@ -0,0 +1,80 @@
+---
+id: overview_zio_prelude_schema
+title: "Use of ZIO Prelude and Schema"
+---
+
+[ZIO Prelude](https://zio.github.io/zio-prelude/docs/overview/overview_index) is a library focused on providing a core set of functional data types and abstractions that can help you solve a variety of day-to-day problems.
+
+### Type-safety with ZIO Prelude's Newtype
+
+Newtypes provide zero overhead new types and refined new types to allow you to increase the type-safety of your code base with zero overhead and minimal boilerplate.
+The library uses ZIO Prelude's Newtype for `IndexName`, `DocumentId`, and `Routing` in order to preserve type-safety and has strings validated when these types are being created.
```

Comment:
We plan on validating strings, but not sure if we should remove it for now as you pointed out.